src/models/description.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const descriptionSchema = new mongoose.Schema({
+  afkorting: String,
+  betekening: String,
```

Comment:
`betekenis` lijkt mij een iets beter woord, een betekening is een gerechtelijke uitspraak... just so you know :)